test/jdk/java/net/httpclient/HttpResponseConnectionLabelTest.java

```diff
@@ -28,9 +28,12 @@
  *          /test/jdk/java/net/httpclient/lib
  * @build jdk.httpclient.test.lib.common.HttpServerAdapters
  *        jdk.test.lib.net.SimpleSSLContext
- * @run junit HttpResponseConnectionLabelTest
+ * @run junit/othervm HttpResponseConnectionLabelTest
  */
 
+// `othervm` in `@run` is intentionally introduced, don't remove it!
+// It helps with, in case `ForkJoinPool.commonPool()` gets corrupt by the test, reusing it again.
```

Comment:
jtreg allows for `@comment` tag in the test definition. So you can replace this newly added code comment with something like:
```
@comment othervm usage is intentional since this test uses ForkJoinPool.commonPool()
```

just before the `@run ....` line in the test definition